java/java.completion/test/unit/data/goldenfiles/org/netbeans/modules/java/completion/JavaCompletionTaskTest/1.8/CreateConstructorNonDefaultConstructor.pass

```diff
@@ -25,9 +25,11 @@ int
 interface
 long
 native
+non-sealed
```

Comment:
I think I would prefer to only have the keywords for an appropriate source level. The existing tests are probably not setting source level to 14, so I would incline to not add the keywords there. New tests may set source level to 14 and check the keywords are proposed by the code completion.